docs/upgrading/upgrading_v3.md

```diff
@@ -233,6 +233,162 @@ const crawler = new BasicCrawler({
 });
 ```
 
+### How does `sendRequest` work?
+
+The `sendRequest` function looks like this:
```

Comment:
we should have this documented elsewhere too, maybe a separate guide mentioning how to work with gotScraping, `sendRequest` helper would be one section there. then the upgrading guide could just link it, right now it feels like this change takes quite a substantial space on the upgrading guide :]

Reply:
created an issue about this